### DISCUSSION TITLE
Is there any chance to generate a session id without any manual clicking?

### DISCUSSION THREAD

**Discussion by lznbb** [GMOD/jbrowse-components#2770] [Q&A] [answered]
The scenario is that we would like to deploy a jbrowse server and upload several related genome/track files by automation. The issue is we just want the session to share across users to come out as long as the configuration is done without any manual operation. In this way, we can get a workflow with high automation level. However, I did not find a jbrowse command to deal with this case. I wonder if there is any possibility to achieve this, or we can never get the session id without clicking manually?
Thanks!

**Comment by cmdcolin** [comment, marked as answer]
hi there. can you clarify the workflow you want to get?
one possible option is that we have a new ability (since 1.6.4) to load sessions using a simplified URL format

some docs here
https://jbrowse.org/jb2/docs/urlparams/

example
http://yoursite/jb2/?loc=chr1:500-600&tracks=trackid1,trackid2&assembly=hg19

there are other options detailed on that page too, but that is the "simplest format"

**Reply by lznbb** [reply]
Thank you very much! I believe that's a suitable solution for me!
